test/numerical/TestGemm.cpp

```diff
@@ -92,10 +93,15 @@ int main(int argc, char *argv[]) {
       const auto aTrans = *rc::gen::inRange(0, 2);
       const auto bTrans = *rc::gen::inRange(0, 2);
       const auto cRank = *rc::gen::inRange(1, 3);
+#ifdef TEST_ALPHA_BETA_1
```

Comment:
Suggest to use a more specific name, say `TEST_GEMM_ALPHA_BETA_1`.

Reply:
Done. Thanks!